app/javascript/components/DownloadButton.js

```diff
@@ -1,25 +1,21 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faDownload } from '@fortawesome/free-solid-svg-icons';
 import Modal from 'react-bootstrap/lib/Modal';
+import Clipboard from 'react-clipboard.js';
 
```

Comment:
webpacker.yml is set up to look to  resolve imports from either app/assets or app/javascript.  So you can do this import as` import { fetchFacetsFilters } from 'lib/scp-api'` .  I'd like to avoid relative imports except between closely-related files in the same directory

Reply:
Addressed in 3bbaa15.